private/migrate/versions_test.go

```diff
@@ -89,9 +89,30 @@ func basicMigration(t *testing.T, db *sql.DB, testDB migrate.DB) {
 		},
 	}
 
+	dbVersion, err := m.CurrentVersion(nil, testDB)
+	assert.NoError(t, err)
+	assert.Equal(t, dbVersion, -1)
+
 	err = m.Run(zap.NewNop())
 	assert.NoError(t, err)
 
+	dbVersion, err = m.CurrentVersion(nil, testDB)
+	assert.NoError(t, err)
+	assert.Equal(t, dbVersion, 2)
+
+	m2 := migrate.Migration{
+		Table: dbName,
+		Steps: []*migrate.Step{
+			{
+				DB:      testDB,
+				Version: 3,
+			},
+		},
+	}
+	dbVersion, err = m2.CurrentVersion(nil, testDB)
+	assert.NoError(t, err)
+	assert.Equal(t, dbVersion, 2)
```

Comment:
Why is this version still 2 after the migration to version 3?

Reply:
same as above?

Reply:
the migration `m2` has a step 3, but we haven't run the migration yet, so we would expect the dbversion to still be 2